### PR TITLE
Yoga classes

### PR DESCRIPTION
**Pull request by emilyPhee**
- Add class list component.
- Add class component containing image, name, duration and level.
- Display yoga class data from Sanity.

### CODE REVIEW

**Comment by emilyPhee** [comment]
[Home Page: Yoga Classes Section](https://trello.com/c/nMqRnax4/13-home-page-yoga-classes-section)

File: nextjs-client/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SANITY_PROJECT_ID=x3fpp54s
+SANITY_DATASET=production
```
Comment (anthopark) [inline nextjs-client/.env]: make sure to gitignore this file

**Review by anthopark — CHANGES_REQUESTED** [review]

**Review by emilyPhee — COMMENTED** [review]
reviewed

**Review by anthopark — APPROVED** [review]
Great job!

**Pull request merged**